### PR TITLE
Replace CP headers with hero

### PR DESCRIPTION
**Pull request by LidorLapid**

### CODE REVIEW

File: _includes/elements/hero.html [hunk 1]
```diff
@@ -1,39 +1,40 @@
...
+{% if include.sliderItems.size > 1 %}
   {% assign isSliderActive = true %}
 {% else %}
   {% assign isSliderActive = false %}
 {% endif %}
 
 <div class="row">
   <div class="column">
-    <section class="hero {% if this.type %}{{ this.type }}{% endif %}{% if isSliderActive %} hero--slider{% endif %}{% if this.desktopStrapline %} hero--strapline{% endif %} row-bust">
+    <section class="hero {% if include.type %}{{ include.type }}{% endif %}{% if isSliderActive %} hero--slider{% endif %}{% if include.desktopStrapline %} hero--strapline{% endif %} row-bust">
```
Comment (wellhairy) [inline _includes/elements/hero.html]: Any reason we've change the include params from this back to include? We use this as standard across the rest of our jekyll files so this seems like something that shouldn't be part of this branch and should be applied more globally if thats the direction we're taking.

File: _includes/elements/slider.html [hunk 2]
```diff
@@ -1,10 +1,12 @@
-{% assign this = include.params | default: include %}
-
-<div class="slider">
-  {% for sliderItem in this.sliderItems %}
+<div class="slider{% if include.type %} {{include.type}}{% endif %}">
```
Comment (wellhairy) [inline _includes/elements/slider.html]: See previous comment about the use of this and include.

File: assets/scss/partials/_variables.scss [hunk 3]
```diff
@@ -73,6 +73,7 @@ $aspect-ratio-square: 100%; // 100 : 100 also 1 : 1 or 100%
 $aspect-ratio-photo: 100% / 3 * 2; // 1920 : 1280 also 1.5 : 1 or 66.66%
 $aspect-ratio-widescreen: 100% / 16 * 9; // 1920 : 1080 also 1.78 : 1 or 56.25%
 $aspect-ratio-super-widescreen: 100% / 18 * 9; // 1920 : 960 also 2 : 1 or 50%
+$aspect-ratio-strip: 40%;
```
Comment (wellhairy) [inline assets/scss/partials/_variables.scss]: What does this do? it's also not exactly a ratio.

File: assets/scss/partials/elements/_navigation-bar-signposts.scss [hunk 4]
```diff
@@ -5,26 +5,43 @@
   background-color: rgba($overcast-teal, 0.9);
 
   &__item-list {
+    @include clearfix;
```
Comment (wellhairy) [inline assets/scss/partials/elements/_navigation-bar-signposts.scss]: Is this necessary now? it wasn't prior to these changes?

File: assets/scss/partials/elements/_slider.scss [hunk 5]
```diff
@@ -3,92 +3,133 @@
...
   @include breakpoint(large) {
     padding-bottom: 0;
   }
 
   &__item {
-    height: 100%;
     width: 100%;
-    background-size: cover;
-    transition: opacity 1s ease-out;
+    height: 101%;
```
Comment (wellhairy) [inline assets/scss/partials/elements/_slider.scss]: whats going on here?

File: assets/scss/partials/headings/_hero.scss [hunk 6]
```diff
@@ -148,7 +147,7 @@ standard content editable hero variation
   &--tall {
     #{$this}__inner {
       @include breakpoint(medium) {
-        min-height: 700px;
+        min-height: 800px;
```
Comment (wellhairy) [inline assets/scss/partials/headings/_hero.scss]: Can we check the difference between 700px and 800px on a standard KCC monitor please. Its my concern that one will fit in a standard viewport and one wont

File: assets/scss/partials/headings/_hero.scss [hunk 7]
```diff
@@ -87,7 +87,6 @@ standard content editable hero variation
   }
 
   &__content {
-    height: 100%;
```
Comment (wellhairy) [inline assets/scss/partials/headings/_hero.scss]: is this not needed? im sure there must be elements inside hero that depend on this line?

File: assets/scss/partials/elements/_slider.scss [hunk 8]
```diff
@@ -3,92 +3,133 @@
   padding-bottom: $aspect-ratio-photo;
   position: relative;
 
-  @include breakpoint(medium) {
-    padding-bottom: $aspect-ratio-widescreen;;
+  &--strip {
+    padding-bottom: $aspect-ratio-strip;
   }
```
Comment (wellhairy) [inline assets/scss/partials/elements/_slider.scss]: there should be a better naming convention for this. I dont really know what strip is.

File: assets/scss/partials/elements/_slider.scss [hunk 9]
```diff
@@ -3,92 +3,133 @@
...
-    background-size: cover;
-    transition: opacity 1s ease-out;
+    height: 101%;
     position: absolute;
-    opacity: 0;
     top: 0;
     z-index: 1;
+    opacity: 0;
+    background-size: cover;
+    background-position: center bottom;
```
Comment (wellhairy) [inline assets/scss/partials/elements/_slider.scss]: Can this not just be background position center center so we can remove line 29 to 31

File: assets/scss/partials/elements/_navigation-bar-signposts.scss [hunk 10]
```diff
@@ -5,26 +5,43 @@
...
+    padding: 11px 14px;
     border: 2px solid $deep-sea-teal;
     border-radius: 3px;
     color: $deep-sea-teal;
     font-weight: 600;
 
+    @include breakpoint(medium) {
+      margin-right: 17px;
+      padding: 10px 17px 11px;
+      font-size: $text-size-medium;
```
Comment (wellhairy) [inline assets/scss/partials/elements/_navigation-bar-signposts.scss]: It'd be nice if going forward we used the consistent font size for buttons

File: assets/scss/partials/elements/_navigation-bar-signposts.scss (hunk 4 above)
Comment (LidorLapid) [inline assets/scss/partials/elements/_navigation-bar-signposts.scss]: It needs it as the list items are being floated. They don't take up the space they need to without clearing the floats.

**Pull request merged**